client.go

```diff
@@ -70,26 +68,25 @@ type Auth struct {
 }
 
 // NewNativeClient creates a new Client using the golang ssh library
-func NewNativeClient(user, host, clientVersion string, port int, auth *Auth) (Client, error) {
-	if clientVersion == "" {
-		clientVersion = "SSH-2.0-Go"
+func NewNativeClient(user, hostname string, clientVersion []byte, auth *Auth) (Client, error) {
```

Comment:
I'm not a fan of docker's go code for this reason. Lets create a new exported function with a new signature so we don't break things using this function